tests/e2e/playback/controls/player/button-widget.e2e.js

```diff
@@ -0,0 +1,103 @@
+  .page // declare the fixture
+`http://localhost:8080/playback/controls/player/button-widget/basic-button`; // specify the start page
+
+test("testing the button Widget basic display", async (t) => {
+  const screenshot_path_base = path.join(
+    "playback",
+    "controls",
+    "player",
+    "button-widget"
+  );
+  const screenshot_path = path.join(
+    screenshot_path_base,
+    "basic-button-widget"
+  );
+  await testcafe_page_visual_regression(t, screenshot_path);
+});
+
+test("testing the ButtonWidget and hover on the buttons", async (t) => {
+  const screenshot_path_base = path.join(
+    "playback",
+    "controls",
+    "player",
+    "button-widget"
+  );
+
+  const buttons = ["cancel", "delete", "save"];
+  var count = await span__button_label.count;
+
+  for (var i = 0; i < count; i++) {
+    let screenshot_path = path.join(
+      screenshot_path_base,
+      "button-widget-hover-" + buttons[i]
+    );
+    await t.hover(span__button_label.nth(i));
+    await testcafe_page_visual_regression(t, screenshot_path);
+  }
+});
+
+fixture`playback/controls/player/button-widget/button-grey`
+  .page // declare the fixture
+`http://localhost:8080/playback/controls/player/button-widget/button-grey`; // specify the start page
+
+test("testing the button Widget basic display with Greyed button", async (t) => {
+  const screenshot_path_base = path.join(
+    "playback",
+    "controls",
+    "player",
+    "button-widget"
+  );
+  const screenshot_path = path.join(
+    screenshot_path_base,
+    "button-grey-display"
+  );
+  await testcafe_page_visual_regression(t, screenshot_path);
+});
+
+fixture`playback/controls/player/button-widget/multiple-button`
+  .page // declare the fixture
+`http://localhost:8080/playback/controls/player/button-widget/multiple-button`; // specify the start page
+
+test("testing the multiple options capability using Button Widget with an X-Y Offset", async (t) => {
+  const screenshot_path_base = path.join(
+    "playback",
+    "controls",
+    "player",
+    "button-widget"
+  );
+  const screenshot_path = path.join(screenshot_path_base, "multiple-button");
+  await testcafe_page_visual_regression(t, screenshot_path);
+});
+
+fixture`playback/controls/player/button-widget/button-event`
+  .page // declare the fixture
+`http://localhost:8080/playback/controls/player/button-widget/button-event`; // specify the start page
+test("testing the ButtonWidget and hover on the buttons", async (t) => {
+  const screenshot_path_base = path.join(
+    "playback",
+    "controls",
+    "player",
+    "button-widget"
+  );
+
+  const buttons = ["cancel", "delete", "save"];
+  var count = await span__button_label.count;
+
+  for (var i = 0; i < count; i++) {
+    let screenshot_path = path.join(
+      screenshot_path_base,
+      "button-widget-Clicked-" + buttons[i]
+    );
+    await t.click(span__button_label.nth(i));
+    await testcafe_page_visual_regression(t, screenshot_path);
```

Comment:
Visual screenshots take much longer to run than other types of tests, so we should only use them when we need to validate some visual aspect of our component.  Testcafe has a variety of tools that allow you to assert that the DOM element in the page updated after you clicked the button. Please use that approach here instead

Reply:
Modified to just asset the contents of the div and with expect statement verified if its matching no VRT included now.

So the CI  on Image comparison time consuming aspect is not impacted, so just text level asserts now.